packages/site/.env

```diff
@@ -1 +1,2 @@
-SNAP_ORIGIN=local:http://localhost:8080
+# SNAP_ORIGIN=local:http://localhost:8080
+GATSBY_SNAP_ORIGIN=npm:@bobanetwork/snap-account-abstraction-keyring-hc
```

Comment:
any reason we changed the name? (gotta update the readme)

Reply:
the above one is not getting available in gatsby so to use in web or browser we need to add `GATSBY_` prefix to env var.